packages/fluentui/react-northstar/src/components/Chat/ChatMessageHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import Box, { BoxProps, BoxStylesProps } from '../Box/Box';
+
+interface ChatMessageHeaderOwnProps {}
+export interface ChatMessageHeaderProps extends ChatMessageHeaderOwnProps, BoxProps {}
+
+export type ChatMessageHeaderStylesProps = never;
+export const ChatMessageHeaderClassName = `ui-chat__message__header`;
+
+/**
+ * A ChatMessageHeader provides a title for the Attachment.
+ */
+const ChatMessageHeader = compose<
+  'span',
+  ChatMessageHeaderOwnProps,
+  ChatMessageHeaderStylesProps,
+  BoxProps,
+  BoxStylesProps
+>(Box, {
+  className: ChatMessageHeaderClassName,
+  displayName: 'ChatMessageHeader',
+
+  overrideStyles: true,
```

Comment:
```suggestion
  shorthandConfig: { mappedProp: 'content' },
  overrideStyles: true,
```

Reply:
`shorthandConfig: { mappedProp: 'content' }` it throws complaining it conflicts with `children`

Reply:
I pushed a fix, now should work 🚀

Reply:
The error seems to still `Warning: Failed prop type: Prop `content` in `ChatMessageHeader` conflicts with props: `children`. They cannot be defined together, choose one or the other.`

Reply:
Fixed using like this:

```jsx
    overrideProps: () => ({
      content: (
        <>
          {authorElement}
          {timestampElement}
          {reactionGroupPosition === 'start' && reactionGroupElement}
        </>
      ),
    }),
```